pkg/build/strategies/sti/sti.go

```diff
@@ -218,6 +219,17 @@ func (b *STI) PostExecute(containerID, location string) error {
 
 	buildEnv := append(scripts.ConvertEnvironment(env), b.generateConfigEnv()...)
 
+	labels := map[string]string{}
+	if b.sourceInfo != nil {
+		labels = map[string]string{
+			"io.openshift.build.commit.author":     b.sourceInfo.Author,
```

Comment:
i don't want to have this configurable because it makes things much harder for no good reasons.